### ISSUE TITLE
Accuracy of aagcn

### ISSUE THREAD

**Issue by ilikeokoge** [lshiwjx/2s-AGCN#41]
I ran your implemented code using J-AAGCN and NTU-RGBD CV dataset.
But Accuracy is 94.64, not 95.1 in your paper.
What is the difference?
The batch size was 32, not 64 because of the resource limit.
Are there any other things to be aware of? I use your implemented code.

**Comment by desertfex** [comment]
Different hardwares lead to different accuracies. Tuning the learning rate and using gradient accumulation may achieve the performance.

**Comment by ilikeokoge** [comment]
Thanks for your comment!

**Comment by dangao250** [comment]
> 
> 
> I ran your implemented code using J-AAGCN and NTU-RGBD CV dataset.
> But Accuracy is 94.64, not 95.1 in your paper.
> What is the difference?
> The batch size was 32, not 64 because of the resource limit.
> Are there any other things to be aware of? I use your implemented code.
Have you successfully ran out of 95.1? How to set your learning rate, can you ask?

**Comment by dangao250** [comment]
> I ran your implemented code using J-AAGCN and NTU-RGBD CV dataset.
> But Accuracy is 94.64, not 95.1 in your paper.
> What is the difference?
> The batch size was 32, not 64 because of the resource limit.
> Are there any other things to be aware of? I use your implemented code.
Can AAGCN's attention explain why convolution can be used and the residual operation code is not reflected. Can you talk about the process? Why does the paper here say that it is a feature map, an attention map, and a graph convolution operation.
